frontend/components/token-classifier/results/entityHighlight.spec.js

```diff
@@ -1,8 +1,30 @@
 import { shallowMount } from "@vue/test-utils";
 import EntityHighlightComponent from "./EntityHighlight";
+import { mockViewSettingsByDatasetName } from "@/models/__mocks__/view-settings/viewSettingsUtilities.mock";
```

Comment:
Do we need a separate file for this mock? Could we just include the mock data in `jest.mock` method?

```javascript
jest.mock("@/models/viewSettings.queries", () => ({
  getViewSettingsByDatasetName: () => {
  $id: "medical-keywords",
  id: "medical-keywords",
  pagination: null,
  viewMode: "annotate",
  loading: false,
  headerHeight: 288,
  visibleMetrics: false,
  visibleRulesList: false,
},
}));
```

Reply:
Since query functions are used in several places, and somethimes could be more big than this one (see dataset query function), I create this file to initiate the structure for the other one  (means I will replace later for the other unit test files). But if it's better I can keep all the mocked data inside their test file

Reply:
For me, it makes sense if the same mock will be reused by several places. If it's used locally, I think is easier to understand and maintain if the mock keeps close to the test.